migrations/0019_wallet_card_id.down.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE wallets DROP COLUMN user_deposit_card_id;
```

Comment:
`card_id` is pretty uphold specific. should we use something like `user_deposit_destination`? or similar

Reply:
https://github.com/brave-intl/bat-go/pull/469/commits/3b24f7861df843e71c37ca657745785d6b2730c5